tests/nosetests/pyanaconda_tests/module_security_test.py

```diff
@@ -173,30 +175,20 @@ def realm_kickstart_test(self):
     def realm_discover_default_test(self, publisher):
         """Test module in default state with realm discover task."""
         realm_discover_task_path = self.security_interface.DiscoverRealmWithTask()
-
-        publisher.assert_called()
```

Comment:
Just wondering aloud: All the `assert_called` were redundant?

Reply:
Yes, they are. The publisher is checked in `check_task_creation`.